bin/q.js

```diff
@@ -1,12 +1,84 @@
 #!/usr/bin/env node
 
-const program = require("commander");
+const { program } = require("commander");
+const chalk = require("chalk");
+const errorColor = chalk.red;
 const version = require("../package.json").version;
+const runServer = require("./commands/server.js");
+const newToolOrServer = require("./commands/newToolOrServer.js");
+const updateItem = require("./commands/updateItem/updateItem.js");
 
-program
-  .version(version)
-  .description("Q Toolbox cli")
-  .command("new-tool", "bootstrap a new tool")
-  .command("new-server", "bootstrap a new Q server implementation")
-  .command("server", "start a simple server mocking a Q server for development")
-  .parse(process.argv);
+async function main() {
+  program.version(version).description("Q Toolbox cli");
+
+  program
+    .command("server")
+    .option("-p, --port [port]", "the port to start the server on")
+    .option(
+      "-t, --target [target]",
+      "the target being used for rendering info route"
+    )
+    .option(
+      "-b, --tool-base-url [url]",
+      "the tool base url being set in rendering info route"
+    )
+    .option(
+      "-c, --config [config]",
+      "the config file name in which additional rendering info and tool runtime config is specified"
+    )
+    .description("start a simple server mocking a Q server for development")
+    .action(async () => {
+      await runServer(program);
```

Comment:
same here, the options are stored in command object instead of program object, so you should pass `commandObject` to `runServer` method

Reply:
Fixed in https://github.com/nzzdev/Q-cli/pull/30/commits/901e5d9c606a50feaee1bd1df5a7d2cb0affd04a